### ISSUE TITLE
plugin-server  - 1.36 possible regression during upgrade

### ISSUE THREAD

**Issue by guidoiaquinti** [PostHog/posthog#10164]
## Bug description
I am investigating a possible regression of `plugin-server` and/or our ingestion pipeline during the upgrade to PostHog version 1.36. See [CI run](https://github.com/PostHog/charts-clickhouse/runs/6689682463?check_suite_focus=true) from our Helm chart repo (we also had a couple of reports from customers).

## How to reproduce
I was not able to reproduce this issue. It seems to be happening only on GitHub actions (maybe due to limited resources?).

Few additional notes:

1. the upgrade path seems to be broken from `1.35` to `1.36` as well as from `1.36` to `1.36`
2. the upgrade path does not seems to be broken from `1.35` to `latest` or `1.36` to `latest`

The suspicious is that the bug/regression has been fixed between `1.36` and `latest` 

## Environment

- [ ] PostHog Cloud
- [X] self-hosted PostHog (ClickHouse-based), version/commit: _please provide_
- [ ] self-hosted PostHog (Postgres-based, legacy), version/commit: _please provide_

## Additional context
* Internal [Slack thread](https://posthog.slack.com/archives/C01MM7VT7MG/p1654151980708239)

**Comment by guidoiaquinti** [comment]
I added some logging in CI and I can't see anything wrong in the `plugin-server` logs, the only anomaly seems to be 

```
(node:35) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 consumer.fetch_start listeners added to [EventEmitter]. Use emitter.setMaxListeners() to increase limit
(Use `node --trace-warnings ...` to show where the warning was created)
```
that was fixed via https://github.com/PostHog/posthog/pull/10097

**Comment by guidoiaquinti** [comment]
[We've released 1.36.1](https://github.com/PostHog/posthog/pull/10217) to address this issue
